ocabundles.json

```diff
@@ -1,6 +1,6 @@
 {
 "4DsheHjrS8kbzw4Rd8Peym:2:Digital Business Card:1.0.0": { "path": "OCABundles/schema/bcgov-digital-trust/business-card/OCABundle.json" },
-"AuJrigKQGRLJajKAebTgWu:3:CL:209526:default": { "path": "OCABundles/schema/bcgov-digital-trust/member-card/OCABundle.json" },
+"AuJrigKQGRLJajKAebTgWu:3:CL:209526:default": { "path": "OCABundles/schema/bcgov-digital-trust/member-card-showcase/OCABundle.json" },
```

Comment:
This change appears to be incorrect. When I run the ./scripts/gen_ocabundlesjson.sh the `AuJrigKQGRLJajKAebTgWu` Identifier has the wrong value -- should be `member-card`, but in this PR it is `member-card-showcase`.  Perhaps this was changed in the file vs. generated?

Reply:
I could rename the `showcase` bundles to be `non-prod` or `staging` since we're going to use them for more than the showcase now

Reply:
ah okay got it, forgot about the readme

Reply:
Okay I've updated the READMEs and can confirm the ocabundles.json generation script works as intended